Pull request template

stefdworschak opened the pull request:
**Description:**
Adding a template for Pull Requests with suggested headings to simplify the process of creating a new Pull Request.

TravelTimN commented:
@stefdworschak are you waiting for me on this one?

stefdworschak commented:
Yep. Please review and if you're okay with it approve it.

TravelTimN commented on static/documentation/forking.md:
```diff
@@ -0,0 +1,38 @@
+
+## Forking the repo
+
+In order to contribute to this project you will need to create a fork of the repo on your own account. 
```
Missing comma:
**In order to contribute to this project, you will need to create a fork of the repo on your own account.**

stefdworschak replied:
Updated as suggested.

TravelTimN commented on static/documentation/forking.md:
```diff
@@ -0,0 +1,38 @@
+## Forking the repo
+
+In order to contribute to this project you will need to create a fork of the repo on your own account. 
+To fork the repo just open the repository in a browser and click on the "fork" button on the top right of the page.
```
Missing comma:
**To fork the repo, just open the repository in a browser and click on the "fork" button on the top right of the page.**

stefdworschak replied:
Updated as suggested.

TravelTimN commented on static/documentation/forking.md:
```diff
@@ -0,0 +1,38 @@
+In order to contribute to this project you will need to create a fork of the repo on your own account. 
+To fork the repo just open the repository in a browser and click on the "fork" button on the top right of the page.
+
+If you are part of any organisations it will ask you which organisation you want to create the fork in.
```
Missing comma:
**If you are part of any organisations, it will ask you which organisation you want to create the fork in.**

stefdworschak replied:
Updated as suggested.

TravelTimN commented on static/documentation/forking.md:
```diff
@@ -0,0 +1,38 @@
+### Gitpod
+
+1. Open your forked repo in the browser
+2. Click on the GitPod button to open a new workspace
```
Gitpod* lowercase `p`, not like GitHub capital `H`

stefdworschak replied:
Updated as suggested.

TravelTimN commented on static/documentation/forking.md:
````diff
@@ -0,0 +1,38 @@
+```
+
+8. Go back to your forked repo in the browser
+9. Click on the message saying "
````
Is there something missing here?
9 just ends with:

`9. Click on the message saying "`

stefdworschak replied:
Added rest of the file.

TravelTimN commented on pull_request_template.md:
```diff
@@ -0,0 +1,24 @@
+**Testing:**
+
+Explain what kind of testing was performed and list any
+testing instructions (if needed)
```
Does this need to be a separate line?

stefdworschak replied:
Yep. For some reason this file was added in without me noticing it. Had planned to add it to another PR, but yeah.
Added in the rest of the file.

TravelTimN reviewed (changes requested):
Minor grammatical adjustments, and then the end of the file seems to be missing on the `forking.md`.

stefdworschak commented:
@TravelTimN thanks for reviewing. Could you please have another look. I added the rest of the forking instructions. Also, the images are linked already, but will only be working once the PR is merged as I have added the link to where the files will be on master.

TravelTimN reviewed (approved):
LGTM 👍
